.github/ISSUE_TEMPLATE/troubleshooting-question.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Troubleshooting question
+about: Ask a question to solve installation and configuration issues
+
+---
+
+For troubleshooting, see [https://www.mattermost.org/troubleshoot/](https://www.mattermost.org/troubleshoot/).
```

Comment:
It seems weird to have a issue template if we don't want community members to use it. Should we use [custom links](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) and link directly instead?